controller/user_test.go

```diff
@@ -248,6 +256,13 @@ func (m TestUserRepository) Load(ctx context.Context, id uuid.UUID) (*account.Us
 	return m.User, nil
 }
 
+func (m TestUserRepository) Exists(ctx context.Context, id string) (bool, error) {
+	if m.User == nil {
+		return false, errors.New("not found")
```

Comment:
Please use `errs.New("not found")` in order to have more information on the error.